### PR TITLE
(DOCS-4112) Add NetFlow beta docs

### PR DESCRIPTION
**Pull request by jtappa**
### What does this PR do?
Adds NetFlow monitoring beta docs

### Motivation
Small private beta available/being announced

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/network_monitoring/devices/netflow.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: NetFlow Monitoring
+kind: documentation
+further_reading:
+- link: "/network_monitoring/devices/profiles"
+  tag: "Documentation"
+  text: "Using Profiles with Network Device Monitoring"
+---
+
+{{< beta-callout url="#" btn-hidden="true" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
```
Comment (hestonhoffman) [inline content/en/network_monitoring/devices/netflow.md:10]:
```suggestion
{{< beta-callout url="#" btn_hidden="true" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
```

File: content/en/network_monitoring/devices/netflow.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: NetFlow Monitoring
+kind: documentation
```
Comment (hestonhoffman) [inline content/en/network_monitoring/devices/netflow.md:3]:
```suggestion
kind: documentation
is_beta: true
```
We've been adding is_beta: true to our other docs. It no longer stops the page from being indexed. I think there might be some other functionality linked to it, like an icon or something, that the web team is going to fix for us.

File: content/en/network_monitoring/devices/netflow.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Installation
+
+To use NetFlow Monitoring with Network Device Monitoring, ensure you are using the [Agent][1] version 7.40 or newer.
+
+**Note:** Configuring [metric collection from Network Device Monitoring][2] is not a requirement for sending NetFlow data, although it is recommended.
+
+## Configuration
+
+To configure your devices to send NetFlow, sFlow, or IPFIX traffic to Agent NetFlow Server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.
```
Comment (hestonhoffman) [inline content/en/network_monitoring/devices/netflow.md:26]:
```suggestion
To configure your devices to send NetFlow, sFlow, or IPFIX traffic to Agent NetFlow Server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.
```
Should this be "the/an" Agent NetFlow Server? Or is there only one thing called Agent NetFlow Server where all the traffic goes?

**Review by hestonhoffman — APPROVED** [review]
Couple small changes, but LGTM!

File: content/en/network_monitoring/devices/netflow.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+## Installation
+
+To use NetFlow Monitoring with Network Device Monitoring, ensure you are using the [Agent][1] version 7.40 or newer.
+
+**Note:** Configuring [metric collection from Network Device Monitoring][2] is not a requirement for sending NetFlow data, although it is recommended.
+
+## Configuration
+
+To configure your devices to send NetFlow, sFlow, or IPFIX traffic to the Agent NetFlow Server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.
```
Comment (jtappa) [inline content/en/network_monitoring/devices/netflow.md:27]:
```suggestion
To configure your devices to send NetFlow, sFlow, or IPFIX traffic to the Agent NetFlow server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.
```

File: content/en/network_monitoring/devices/netflow.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+is_beta: true
+further_reading:
+- link: "/network_monitoring/devices/profiles"
+  tag: "Documentation"
+  text: "Using Profiles with Network Device Monitoring"
+---
+
+{{< beta-callout url="#" btn_hidden="true" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
+NetFlow Monitoring for Datadog Network Device Monitoring is in private beta.
+{{< /beta-callout >}}
```
Comment (jtappa) [inline content/en/network_monitoring/devices/netflow.md:13]:
```suggestion
<div class="alert alert-warning">NetFlow Monitoring for Datadog Network Device Monitoring is in private beta.</div>
```

**Pull request merged**